README.md

```diff
@@ -27,3 +27,78 @@ You will need to configure the following environment variables and make them ava
 Also, for publishing to GitHub, you will need to provide a GitHub API key.
 
 For that, in travis, you can use the `env`. If your project is public, make sure to use [secrets](https://docs.travis-ci.com/user/encryption-keys/).
+
+## Tasks
+
+### Tests packages
+
+Three rules are available `test` plain execution of test, `test-race` execute
+the test with the `-race` flag and `test-coverage` it runs the tests with
+coverage support and generate the `coverage.txt` file. The coverage file can
+be uploaded to codecov using the rule `codecov`.
+
+### Building packages
+
+The rule `packages` creates the distribution packages, containing the command
+line utilities defined by the `COMMANDS` compiled for the architectures and
+OS, define by `PKG_OS` and `PKG_ARCH`.
+
+The package filename is based on the pattern: `<project>_<version>_<os>_<arch>.tar.gz`
+
+### Docker build and push
+
+The rule `docker-build` builds the defined dockerfiles by `DOCKERFILES`. Several
+dockerfile may be define per project eg.:
+
+If we have an application with two different docker images, can be archived
+creating two different docker files one call `Dockerfile.server` and
+`Dockerfile.client`.
+
+```
+DOCKERFILES = Dockerfile.server:$(PROJECT)-server Dockerfile.client:$(PROJECT)-client
+```
+
+The `DOCKERFILES` variable should be defined as list of pairs file/name, example `Dockerfile:my-image`
+
+The rule `docker-push` build and push the defined dockerimages. The images are
```

Comment:
s/build and push the defined dockerimages/builds and pushes to the defined registry all the defined docker images/